docs/content/en/docs/reference/clusterspec/optional/registrymirror.md

```diff
@@ -74,7 +74,7 @@ export REGISTRY_PASSWORD=<password>
 ```
 
 ### __insecureSkipVerify__ (optional)
-* __Description__: optional field to skip the registry certificate verification. Only use this solution for isolated testing or in a tightly controlled, air-gapped environment. Currenly only supported for Ubuntu OS.
+* __Description__: optional field to skip the registry certificate verification. Only use this solution for isolated testing or in a tightly controlled, air-gapped environment. Currenly not supported for Bottlerocket OS.
```

Comment:
Does this imply that it is supported for Ubuntu and RHEL?

Reply:
Yes it does

Reply:
Yes.